src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/DependenciesGraphProvider.cs

```diff
@@ -53,11 +53,11 @@ internal class DependenciesGraphProvider : OnceInitializedOnceDisposedAsync, IGr
                                     ImportOrderPrecedenceComparer.PreferenceOrder.PreferredComesLast);
         }
 
-        private readonly GraphCommand ContainsGraphCommand = new GraphCommand(
+        private static readonly GraphCommand[] ContainsGraphCommand = new[] { new GraphCommand(
```

Comment:
It's literally only a single command called "Contains" - I think the name describes it correctly.